lgpo/README.md

```diff
@@ -0,0 +1,89 @@
+# lgpo
+
+This package provides the LGPO binaries
+
+[LGPO](https://blogs.technet.microsoft.com/secguide/2016/01/21/lgpo-exe-local-group-policy-object-utility-v1-0/) is a command-line utility to automate the management of local group policy.
+
+## Maintainers
+
+The Habitat Maintainers humans@habitat.sh
+
+## Type of Package
+
+[binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)
+
+## Usage
+
+LGPO.exe can be used to apply local group policies to a machine using habitat
+
+```
+$ hab pkg install core/lgpo
+$ hab pkg binlink core/lgpo
```

Comment:
we have not implemented binlink on windows. Which reminds me to open an issue for that. In the meantime you could replace this line and the line below with `hab pkg exec core/lgpo LGPO.exe`